appengine/reconciletags/version_check_test.py

```diff
@@ -13,17 +13,17 @@
 # This is the only way to import LooseVersion that will actually work
 from distutils.version import LooseVersion
 
-runtime_to_version_check = {
+runtime_to_latest_version = {
     "aspnetcore": ("git ls-remote --tags https://github.com/dotnet/core"
-                   "| egrep -o \"{}\\.[0-9]+$\" | cut -c 1-"),
+                   "| egrep -o \"{}\\.[0-9]+$\""),
```

Comment:
Why do we still have the egreps and awks in these? I was hoping to remove them and just have a single command, then parse the rest in python.